src/test/java/org/apache/maven/plugins/javadoc/AggregatorJavadocReportTest.java

```diff
@@ -257,4 +249,30 @@ public void testAggregateJavadocResources()
         assertTrue( overview.contains( "<img src=\"doc-files/maven-feather.png\" alt=\"Maven\">" ) );
         assertTrue( new File( apidocs, "resources/test/doc-files/maven-feather.png" ).exists() );
     }
+
+    public void testAggregateWithModulsNotInSubFolders() throws Exception
+    {
+      File testPom = new File( unit, "aggregate-modules-not-in-subfolders-test/all/pom.xml");
+      JavadocReport mojo = lookupMojo( testPom );
+      mojo.execute();
+      
+      File apidocs = new File( getBasedir(), "target/test/unit/aggregate-modules-not-in-subfolders-test/target/site/apidocs" );
+      assertTrue(apidocs.exists());
+      assertTrue( getOverviewSummary(apidocs).exists() );

```

Comment:
`( apidocs )`

Reply:
Seems checkstyle does not run on test classes ...
Will fix